tests/fixtures/resources.py

```diff
@@ -0,0 +1,10 @@
+
+import pytest
+
+
+@pytest.fixture
+def resource0(project0):
+    from pontoon.base.models import Resource
```

Comment:
Why do you import those at runtime?

Reply:
it might not be necessary, but often the fixtures are loaded before django has been configured - ill check

Reply:
turns out this isnt necessary - imports moved